pypeit/core/flexure.py

```diff
@@ -294,12 +294,9 @@ def spec_flex_shift(obj_skyspec, arx_skyspec, arx_lines, mxshft=20, excess_shft=
 
             elif excess_shft == "skip":
                 msgs.warn("Flexure compensation failed for one of your objects.")
-                msgs.warn("Skipping the flexure correction, shift = 0")
-                # Return the usual dictionary, but with a shift == 0
-                return dict(polyfit=fit, shift=0.0, subpix=subpix_grid,
-                            corr=corr[subpix_grid.astype(np.int)],
-                            sky_spec=obj_skyspec, arx_spec=arx_skyspec,
-                            corr_cen=corr.size/2, smooth=smooth_sig_pix)
+                msgs.warn("Skipping 1D spectral extraction.")
```

Comment:
I appreciate the possibility to skip the extraction of the 1D extraction if the flexure correction fails, but I thought that the option to set it to zero was also a good one. Can we have both `skip_extraction` and `set_to_zero` options for `excess_shft`?

Reply:
Have reverted last change, and changed `skip` to `set_to_zero`.